test/e2e/specs/specs-playwright/wp-signup__wpcc.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * @group calypso-release
+ */
+
+import {
+	DataHelper,
+	LoginPage,
+	setupHooks,
+	UserSignupPage,
+	CloseAccountFlow,
+	ReaderPage,
+	SupportComponent,
+	GutenboardingFlow,
+	EmailClient,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe(
+	DataHelper.createSuiteTitle( 'Signup: WordPress.com WPCC/Reader/Support Popover' ),
+	function () {
+		const inboxId = DataHelper.config.get( 'signupInboxId' ) as string;
+		const username = `e2eflowtestingwpcc${ DataHelper.getTimestamp() }`;
+		const email = DataHelper.getTestEmailAddress( {
+			inboxId: inboxId,
+			prefix: username,
+		} );
+		const signupPassword = DataHelper.config.get( 'passwordForNewTestSignUps' ) as string;
+
+		let page: Page;
+
+		setupHooks( ( args ) => {
+			page = args.page;
+		} );
+
+		describe( 'Signup via /start/wpcc', function () {
+			let activationLink: string;
+
+			it( 'Navigate to CrowdSignal WPCC endpoint', async function () {
+				let calypsoBaseURL = DataHelper.getCalypsoURL();
+				calypsoBaseURL =
+					calypsoBaseURL +
+					'/start/crowdsignal/oauth2-name?ref=oauth2&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D34604bf9bcb69d07819b97c65b744925c061ed944e4d30623b9b8cebf1a4ce34%26redirect_uri%3Dhttps%253A%252F%252Fapp.crowdsignal.com%252Fconnect%253Fsource%253Dsignup%2526action%253Drequest_access_token%26wpcom_connect%3D1%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login&oauth2_client_id=978';
```

Comment:
If this is a constant value, should we make it a part of the config object? 

```js
calypsoBaseURL += DataHelper.config.get( 'WPCCAuthRoutePath' );
```

Reply:
Yeah that's probably a good way to go about it.

Reply:
Fixed in c06160a60fea8e7e37c9965e404be4fc1c25e7bf.